package.json

```diff
@@ -114,6 +114,7 @@
     "fabric8-planner": "0.26.3",
     "fabric8-runtime-console": "0.18.10",
     "fabric8-stack-analysis-ui": "0.1.49",
+    "gsap": "1.20.2",
```

Comment:
it is a JavaScript/TypeScript library. If you want to provide the CSS animate code to replace what is done in JS works for me.